### DISCUSSION TITLE
Create timelapse

### DISCUSSION THREAD

**Discussion by duelago** [roleoroleo/sonoff-hack#65] [Q&A] [answered]
Hello!

Love your hack! 

Question: How do I create a snapshot using the camera itself?
I.e. I want to run cron at a certain time and save a high res snapshot to the SD-card. Possible?

 Best regards,
 David

**Comment by duelago** [comment, marked as answer]
Solved it. Found the snapshot script 
/mnt/mmc/sonoff-hack/bin/snapshot > outputwithdatestamp.jpg

**Reply by roleoroleo** [reply]
Yes.
If you want to create snapshot from outside the cam try the webservice.
Check the wiki.
